tests/test_framework/phpbb_functional_test_case.php

```diff
@@ -1433,4 +1433,112 @@ protected function get_hidden_fields($crawler, $url)
 
 		return $file_form_data;
 	}
+
+	/**
+	 * Get HTML of the crawler
+	 * See https://symfony.com/doc/current/components/dom_crawler.html#component-dom-crawler-dumping
+	 *
+	 * @param Symfony\Component\DomCrawler\Crawler $crawler Crawler instance
+	 * @param string $url Request URL
+	 *
+	 * @return array Hidden form fields array
+	 */
+	protected function dump_crawler($crawler)
```

Comment:
Is this needed?

Reply:
Was handy for debugging tests but removed.